index.src.html

```diff
@@ -631,6 +634,8 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
                   // Insert some code here to fall back to a basic login form.
                 }
               });
+            // Alternatively, the website could just copy the credential data into the
+            // form and call submit() on the form.
```

Comment:
I have added a form.submit() example. I agree that it would be great to have the inverse of `PasswordCredential(HTMLFormElement)` but that should be part of a different change.